Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1569,7 +1569,7 @@ def lhegensim2017(fragment,howMuch):
                                '--era':'Run2_HI',
                                '--customise':'RecoLocalTracker/SiStripZeroSuppression/customiseHybrid.addHybridEmulationBeforeRepack',
                                '--processName':'EMULATEHYBRID',    
-                               #'--customise_commands':'\'process.RAWoutput.outputCommands.append(\"drop *_*_*_HLT*\")\'',
+                               '--outputCommands':'\'drop FEDRawDataCollection_*__HLT.\'',
```

Comment:
I was too quick to push here:
please also change this to ```'"drop FEDRawDataCollection_*__HLT"'```
well, the most important part is to drop the dot at after "HLT"

Reply:
uhm, running further, this is still not enough, need to also drop the LHC rawDataCollector
```'"drop FEDRawDataCollection_*__HLT", "drop FEDRawDataCollection_*__LHC"'```
should do it (the test is not finished yet)